source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -189,12 +186,14 @@ requirements:
 
       MongoCollection<Flower> collection = database.getCollection("flowers", Flower.class);
 
-Once you have configured the preceding ``MongoCollection`` instance, you can:
+Once you have configured the preceding ``MongoCollection`` instance, you
+can perform :ref:`CRUD operations <java-crud-operations>` with the POJO
+such as:
```

Comment:
Issue: the list is introduced with a sentence fragment (both in existing text and this change). The style guide mentions "Avoid using fragments to introduce lists." https://www.mongodb.com/docs/meta/style-guide/style/lists/

Suggestion:

... you can perform the following CRUD operations with POJOs:
- Create a document from a POJO
- Retrieve a data in a POJO instance